### ISSUE TITLE
Mysterious page 0

### ISSUE THREAD

**Issue by jacklul** [php-telegram-bot/inline-keyboard-pagination#10]
There seems to be a bug when you have only one item per page, max buttons set to 8 and 9 or more elements in the array.
When you reach page 3 a mysterious page 0 shows up: https://i.imgur.com/nlMfcVm.png

Test code:
```php
<?php

namespace Longman\TelegramBot\Commands\AdminCommands;

use Longman\TelegramBot\Commands\AdminCommand;
use Longman\TelegramBot\Entities\InlineKeyboard;
use Longman\TelegramBot\Entities\ServerResponse;
use Longman\TelegramBot\Request;
use TelegramBot\InlineKeyboardPagination\InlineKeyboardPagination;

class TestCommand extends AdminCommand
{
    public function execute(): ServerResponse
    {
        $this->getCallbackQuery() && parse_str($this->getCallbackQuery()->getData(), $callback_data);
        $current_page = $callback_data['p'] ?? 1;
        $array = [1,2,3,4,5,6,7,8,9];

        $ikp = new InlineKeyboardPagination($array, 'test', $current_page, 1);
        $ikp->setMaxButtons(8, true);
        $ikp->setCallbackDataFormat('c={COMMAND}&p={NEW_PAGE}');

        $data = [
            'chat_id' => $this->getMessage() ? $this->getMessage()->getChat()->getId() : $this->getCallbackQuery()->getFrom()->getId(),
            'text' => $array[$current_page - 1],
            'reply_markup' => new InlineKeyboard(...[$ikp->getPagination()['keyboard']])
        ];

        if ($this->getCallbackQuery() && $message_id = $this->getCallbackQuery()->getMessage()->getMessageId()) {
            return Request::editMessageText($data + [
                'message_id' => $message_id
            ]);
        }

        return Request::sendMessage($data);
    }
}
```

**Comment by noplanman** [comment]
Weirdness! I'll look into this 👓

**Comment by noplanman** [comment]
I have managed to reproduce it in the unit tests. 🙏 PHPUnit!

Will see no what's causing this.

**Comment by noplanman** [comment]
Fixed in 1.1.0
